core/templates/dev/head/css/oppia.css

```diff
@@ -104,7 +104,7 @@ body {
   width: 100%;
 }
 
-.oppia-profile-container, .oppia-dashboard-container {
+.oppia-dashboard-container {
   max-width: 800px;
   margin-left: auto;
```

Comment:
Optional: you can simplify this as follows:

    margin: 0 auto 30px auto;

(assuming margin-top is 0)